addons/advanced_ballistics/stringtable.xml

```diff
@@ -28,92 +28,115 @@
+            <German>Aktiviert die erweiterte Ballistik für alle nicht lokalen Spieler (das zu aktivieren könnte die Leistung während großen Feuergefechten im Multiplayer reduzieren)</German>
         </Key>
         <Key ID="STR_ACE_AdvancedBallistics_alwaysSimulateForGroupMembers_DisplayName">
             <English>Always Enabled For Group Members</English>
             <Polish>Zawsze akt. dla czł. grupy</Polish>
+            <German>Für Gruppenmitglieder immer aktiviert</German>
         </Key>
         <Key ID="STR_ACE_AdvancedBallistics_alwaysSimulateForGroupMembers_Description">
             <English>Always enables advanced ballistics when a group member fires</English>
             <Polish>Aktywuje zaawansowaną balistykę dla wszystkich członków grupy</Polish>
+            <German>Erweiterte Ballistik immer aktiviert lassen, wenn ein Gruppenmitglieder schießt</German>
         </Key>
         <Key ID="STR_ACE_AdvancedBallistics_disabledInFullAutoMod_DisplayName">
             <English>Disabled In FullAuto Mode</English>
             <Polish>Wył. podczas ognia auto.</Polish>
+            <German>Im Aut</German>
         </Key>
         <Key ID="STR_ACE_AdvancedBallistics_disabledInFullAutoMod_Description">
             <English>Disables the advanced ballistics during full auto fire</English>
             <Polish>Dezaktywuje zaawansowaną balistykę podczas ognia automatycznego</Polish>
+            <German>Deaktiviert die erweiterte Ballistik während Vollautomatik</German>
         </Key>
         <Key ID="STR_ACE_AdvancedBallistics_ammoTemperatureEnabled_DisplayName">
             <English>Enable Ammo Temperature Simulation</English>
             <Polish>Symulacja temp. amunicji</Polish>
+            <German>Munitionstemperatur-Simulation aktivieren</German>
         </Key>
         <Key ID="STR_ACE_AdvancedBallistics_ammoTemperatureEnabled_Description">
             <English>Muzzle velocity varies with ammo temperature</English>
             <Polish>Prędkość wylotowa pocisku jest zależna od temperatury amunicji</Polish>
+            <German>Mün</German>
         </Key>
         <Key ID="STR_ACE_AdvancedBallistics_barrelLengthInfluenceEnabled_DisplayName">
             <English>Enable Barrel Length Simulation</English>
             <Polish>Symulacja długości lufy</Polish>
+            <German>Lauflängen-Simulation aktivieren</German>
         </Key>
         <Key ID="STR_ACE_AdvancedBallistics_barrelLengthInfluenceEnabled_Description">
             <English>Muzzle velocity varies with barrel length</English>
             <Polish>Prędkość wylotowa pocisku jest zależna od długości lufy</Polish>
+            <German>Länge des Laufs verändert Mündungsbeschleunigung</German>
         </Key>
         <Key ID="STR_ACE_AdvancedBallistics_bulletTraceEnabled_DisplayName">
             <English>Enable Bullet Trace Effect</English>
             <Polish>Efekt smugi pocisku</Polish>
+            <German>Leuchtspurgeschosse aktivieren</German>
```

Comment:
This is not about tracers. Either treat `bullet trace` as proper noun or write sth. like: `Geschoss Spur Effekt aktivieren`